codec/codec-jackson/src/test/java/com/alipay/sofa/rpc/codec/jackson/generic/GenericService.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.codec.jackson.generic;
+
+/**
+ * @author gujin
+ * Created on 2022/4/7 8:57 下午
+ */
+public interface GenericService<T, R> {
+
+    public R hello(T t);
```

Comment:
Remove the public for code format and remark there do not support method overloading.

Reply:
@EvenLjj fixed